src/pages/Checklist/index.tsx

```diff
@@ -168,78 +177,186 @@ export const Checklist = () => {
+                  defaultMessage="The Beacon chain had its genesis on 01-Dec-2021. It is growing in size over time, and the
+                    introduction of sharding will also increase storage, memory and bandwidth requirements."
                 />
               </Text>
             </li>
             <li className="py5">
               <Text>
-                <FormattedMessage defaultMessage="Note: it may take day(s) to sync the Eth1 client in the worst-case scenario." />
+                <FormattedMessage defaultMessage="SSD storage is required to consistently handle necessary read/write speeds." />
+              </Text>
+            </li>
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="Please be sure to account for enough space on your drive until you run maintenance on your node." />
+              </Text>
+            </li>
+          </ul>
+        </section>
+        <section>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="CPU and RAM" />
+          </Heading>
+          <ul className="sub-checklist-item">
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="Please check with client documentation to ensure the hardware you anticipate using is sufficient and supported." />
               </Text>
             </li>
           </ul>
         </section>
         <section>
           <Heading level={3}>
-            <FormattedMessage defaultMessage="Security" />
+            <FormattedMessage defaultMessage="Internet" />
           </Heading>
+          <ul className="sub-checklist-item">
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="Your internet connection should be reliable and as close to 24/7 as possible without interruption." />
+              </Text>
+            </li>
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="Bandwidth should not be throttled or capped to ensure node stays in sync and ready to validate when called." />
+              </Text>
+            </li>
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="Upload bandwidth is also important, and as of February 2021 this is ~700-800 MB/hour, and anticipated to increase." />
+              </Text>
+            </li>
+            <li className="py5">
+              <Text>
+                <FormattedMessage defaultMessage="Note: syncing your Eth1 client may take a few days in the worst-case scenario" />
+              </Text>
+            </li>
+          </ul>
+        </section>
+        <SectionHeader>
+          <Heading level={3}>
+            <FormattedMessage defaultMessage="Section 2 - Setup" />
+          </Heading>
```

Comment:
Again, maybe a subtitle that explains this checklist is designed to help you while you go through the deposit process.